src/screens/block/block.vue

```diff
@@ -113,10 +118,22 @@ export default {
       } else if (blockHashRegex.test(this.blockId)) {
         this.$store.dispatch('loadBlock', { hash: this.blockId })
       }
+    },
+    takeAction (event) {
+      if (event.srcKey === 'right' && this.block.height < this.latestHeight) {
+        this.$router.push(`/block/${this.block.height + 1}`)
+      }
+      if (event.srcKey === 'left' && this.block.height > 1) {
+        this.$router.push(`/block/${this.block.height - 1}`)
```

Comment:
Will be better to refer to route by name:
```js
this.$router.push({ name: 'Block', params: { blockId: this.block.height - 1 } })
```
It will make easier to change url format in future.